Dockerfile

```diff
@@ -9,6 +9,7 @@ ADD nakadi /nakadi
 RUN find ./nakadi | grep -E "(__pycache__|\.pyc|\.pyo$)" | xargs rm -rf
 
 ADD run.sh /nakadi/run.sh
+ADD scm-source.json /scm-source.json
```

Comment:
Nakadi JVM has means to generate it in gradle file
Here the problem is that dockerfile requires something that can be generated only using stuff from private repository.